src/ComposerNoUpdaterException.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Kisphp;
+
+class ComposerNoUpdaterException extends \Exception
+{
+    protected $message = 'You should not run "composer update" without any arguments. If you know what you\'re doing, prepend COMPOSER_ENV=dev to composer command.';
```

Comment:
I wouldn't call this an environment, just make it an explicit option COMPOSER_UPDATE_FORCE=1